packages/rendermime/src/renderers.ts

```diff
@@ -398,6 +399,12 @@ export function renderMarkdown(
       if (shouldTypeset && latexTypesetter) {
         latexTypesetter.typeset(host);
       }
+      if (fragment) {
+        const el = host.querySelector(fragment);
```

Comment:
Since these are user-settable, I found it pretty easy to accidentally enter an invalid fragment, which threw an error. Maybe this block could be in a try/except statement to catch those (possibly with a console warning)?

Reply:
I agree.